executor/cook/executor.py

```diff
@@ -218,7 +222,46 @@ def get_task_state(exit_code):
         return cook.TASK_FINISHED
 
 
-def manage_task(driver, task, stop_signal, completed_signal, config, stdout_name, stderr_name):
+def set_environment(environment, key, value):
+    """Updates an entry in the environment dictionary.
+
+    Returns
+    -------
+    Nothing.
+    """
+    if key not in environment or environment[key] != value:
+        logging.info('Setting process environment[{}]={}'.format(key, value))
+        environment[key] = value
+
+
+def retrieve_process_environment(config, os_environ):
+    """Prepares the environment for the subprocess.
+    The function also ensures that env[EXECUTOR_PROGRESS_OUTPUT_FILE_ENV]
+    and PROGRESS_OUTPUT_FILE is set to config.progress_output_name in the
```

Comment:
What is `PROGRESS_OUTPUT_FILE`?  It's not set in this function.

Reply:
Removed it, I hadn't updated the documentation after cleaning up the code.